server/container.go

```diff
@@ -16,11 +16,10 @@ import (
 // CreateContainer creates a new container in specified PodSandbox
 func (s *Server) CreateContainer(ctx context.Context, req *pb.CreateContainerRequest) (*pb.CreateContainerResponse, error) {
 	// The id of the PodSandbox
-	podSandboxID := req.GetPodSandboxId()
-	if !s.hasSandbox(podSandboxID) {
-		return nil, fmt.Errorf("the pod sandbox (%s) does not exist", podSandboxID)
+	podSandboxName := req.GetPodSandboxId()
```

Comment:
The kubelet will be passing ID here that we return in status, not the name. So, we need to handle that. Basically, the name will be passed for pod/container on their creates, we return the IDs which is then used in subsequent calls.